src/swarm-tui/Swarm/TUI/Model.hs

```diff
@@ -319,7 +319,8 @@ populateInventoryList (Just r) = do
 
 -- | Command-line options for configuring the app.
 data AppOpts = AppOpts
-  { userSeed :: Maybe Seed
+  { version' :: Maybe (IO ())
```

Comment:
Hmm, I don't like the idea of shoving an `IO ()` into a record of options.  Is there a particular reason this was necessary?  Could we just have `version :: Bool` in the record, and then in `appMain` call the appropriate `IO` action when the `Bool` is `True`?

Reply:
What you suggest was my initial attempt, but then I realized that it entailed moving more things around, so I opted for an `IO ()` in the record.

I've given another try to the `Bool` approach, and it works, but it requires moving a few helpers from `Main.hs` to `Swarm/App.hs`.

Is that ok?

Reply:
Moving stuff around is definitely OK.